OpenRA.Mods.Common/Lint/CheckSequences.cs

```diff
@@ -40,19 +40,18 @@ void Run(Action<string> emitError, Action<string> emitWarning, Ruleset rules, Se
 			var factions = rules.Actors[SystemActors.World].TraitInfos<FactionInfo>().Select(f => f.InternalName).ToArray();
 			foreach (var actorInfo in rules.Actors)
 			{
+				var images = new HashSet<string>();
+
 				// Actors may have 0 or 1 RenderSprites traits
 				var renderInfo = actorInfo.Value.TraitInfoOrDefault<RenderSpritesInfo>();
-				if (renderInfo == null)
-					continue;
-
-				var images = new HashSet<string>()
+				if (renderInfo != null)
 				{
-					renderInfo.GetImage(actorInfo.Value, null).ToLowerInvariant()
-				};
+					images.Add(renderInfo.GetImage(actorInfo.Value, null).ToLowerInvariant());
 
-				// Some actors define faction-specific artwork
-				foreach (var faction in factions)
-					images.Add(renderInfo.GetImage(actorInfo.Value, faction).ToLowerInvariant());
+					// Some actors define faction-specific artwork
+					foreach (var faction in factions)
+						images.Add(renderInfo.GetImage(actorInfo.Value, faction).ToLowerInvariant());
+				}
 
 				foreach (var traitInfo in actorInfo.Value.TraitInfos<TraitInfo>())
```

Comment:
As you removed 

```
if (renderInfo == null)
	continue;
```

this will now run, whenever renderInfo is null, is this desired?

Reply:
Yep, that is the whole point. The image override is applied in line 71-81 and works even without `RenderSpritesInfo`.